lms/djangoapps/ccx/api/v0/views.py

```diff
@@ -479,13 +479,13 @@ def post(self, request):
             )
 
             # Hide anything that can show up in the schedule
-            hidden = 'visible_to_staff_only'
-            for chapter in master_course_object.get_children():
-                override_field_for_ccx(ccx_course_object, chapter, hidden, True)
-                for sequential in chapter.get_children():
-                    override_field_for_ccx(ccx_course_object, sequential, hidden, True)
-                    for vertical in sequential.get_children():
-                        override_field_for_ccx(ccx_course_object, vertical, hidden, True)
+            # hidden = 'visible_to_staff_only'
```

Comment:
Can you please leave a comment why it's hidden and when it's possible to uncomment it?

Reply:
@TamoshaytisV I've updated CCX API views to be consistent with UI views. Custom Field Override provider is used, so we don't need to create CCX overrides to hide some blocks.
See https://github.com/Livit/Livit.Learn.EdX/blob/6904ed324c82f489aebf6dd3c34c7f066a110b31/lms/djangoapps/ccx/views.py#L229-L236

Reply:
Agree, but can you please leave a comment in the code explaining this? Otherwise why do we need to keep commented code at all?

Reply:
done